v1_output/decadal/scream_output.decadal.15minINST_ARM.yaml

```diff
@@ -2,12 +2,12 @@
 ---
 filename_prefix: output.scream.decadal.15minINST_ARM
 iotype: pnetcdf
-Averaging Type: Instant
-Max Snapshots Per File: 96  # one file per day
+averaging_type: Instant
```

Comment:
The values were also changed to snake case. `instant` instead of `Instant`

Reply:
Thank you for your suggestion. It has been updated accordingly.